### PR TITLE
:wrench: README language reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,44 @@
...
+Open _marking-menus_ of arbitrary depth via keyboard shortcuts. \
+Transition seemlessly between the three different selection modes.
 
-* **Point-and-Click:** Select items by clicking on them or anywhere in the corresponding wedges.
-* **Marking-Mode:** Select items by drawing gestures. To do this, click anywhere and drag your mouse while the left button is pressed over an item. As soon as you pause the movement or make a turn, the item will be selected.
-* **Turbo-Mode:** You can also "draw" gestures as long as a modifier key, such as <kbd>Ctrl</kbd>, <kbd>Shift</kbd>, or <kbd>Alt</kbd> is held down **without having to press your mouse button**! This is especially useful when you opened the menu with a shortcut involving such a modifier key. You can just keep it pressed and move the pointer with the mouse or your touch-pad!
+* 🐁 **Point-and-Click:** Select items by clicking them, or anywhere in the corresponding wedges.
+* 🔲 **Marking-Mode:** Select items by drawing gestures. \
+Leftclick anywhere and drag your mouse over an item to do so. \
```
Comment (comradekingu) [inline README.md:20]: Yes?

**Comment by Schneegans** [comment]
Thank you for the contribution! However, I am a bit undecided on what to do with this one. Half of the changes I appreciate, the other half is not about language but rather about style and this is a very subjective matter. For example:

* Why did you truncate most sentences so that they are no sentences anymore? Most of your sentences either miss a verb or a subject. This makes it pretty hard to read and somewhat less fluent...
* Why did you put the gif to the bottom?
* I like the idea of using emoji for the different modes, but why 🐁, 🔲, and 🖱️? Why different mice? What's the square supposed to be?
* Why did you add the link to Weblate at this position? It does not make much sense in the table of contents and one cannot see that there are actually two links in one line.
* Sorry, I do not get the "grumpie menu ♜-ie". Is "tower-ie"? "fort-ie"? "castle-ie"? :smile: 

At this point, I would consider this to be a set of well-intentioned suggestions of which I will happily adopt some, but I would refrain from merging them all in this state.

**Comment by comradekingu** [comment]
@Schneegans All with respect to the flow of information.
Preference for what is important, in the sequence of reading it, from top to bottom.
Nothing went missing.

What exists is a wall of text. This in turn induces stress. It is also harder to get any overview.
To put it in perspective, I have 3840 vertical pixels, and it still scrolls one step at 100% scaling.

Link to Hosted Weblate is for SEO. I think their libre plan also requires it, in point 5 https://hosted.weblate.org/hosting/
♜ is the rook. The other emojis can be replaced when someone finds anything better.
:black_square_button: is a selection in my head.

Give it a go, and then make changes down the road if you remember why. As you forget whose ideas belonged to whom, and what the changes were, it will all settle in.
And/or, try the different variants on someone.

**Comment by Schneegans** [comment]
I adopted some of the suggested changes in 88614fb. Thank you for the constructive ideas!

**Pull request closed without merging**